### DISCUSSION TITLE
How to hide back button of NavigationView?

### DISCUSSION THREAD

**Discussion by skysing** [bdlukaa/fluent_ui#424] [Q&A] [answered]
As follow:
![image](https://user-images.githubusercontent.com/5407786/199675144-cd5b3a91-7f8d-4812-ba9f-c831b803a190.png)

**Comment by bdlukaa** [comment]
What icon, exactly?

**Comment by bdlukaa** [comment, marked as answer]
You can set `NavigationAppBar.automaticallyImplyLeading` to `false`
